Shared/Samples/Edit with branch versioning/EditWithBranchVersioningView.swift

```diff
@@ -0,0 +1,250 @@
+                        switch selectedAction {
+                        case .setUp:
+                            try await model.setUp()
+                            
+                        case .makeVersion:
+                            try await model.createVersion(parameters: versionParameters!)
+                            
+                        case .switchToVersion(let version):
+                            try await model.switchToVersion(named: version)
+                            
+                        case .updateFeature:
+                            try await model.updateFeature()
+                            
+                        case .selectFeature(let screenPoint, let mapPoint):
+                            model.clearSelection()
+                            
+                            // Identifies the feature on the feature layer at the tapped point.
+                            let result = try await mapViewProxy.identify(
+                                on: model.featureLayer,
+                                screenPoint: screenPoint,
+                                tolerance: 10
+                            )
+                            guard let feature = result.geoElements.first as? Feature else { return }
+                            
+                            model.selectFeature(feature)
+                            calloutPlacement = .geoElement(feature, tapLocation: mapPoint)
+                        }
+                    } catch {
+                        self.error = error
+                    }
+                }
+        }
+        .overlay(alignment: .top) {
+            let statusText = switch model.state {
+            case .notLoaded: ""
+            case .loading: "Loading service geodatabase…"
+            case .selectedVersion(let name): "Version: \(name)"
+            case .createdVersion(let name): "Created: \(name)"
+            }
+            
+            Text(statusText)
+                .multilineTextAlignment(.center)
+                .frame(maxWidth: .infinity, alignment: .center)
+                .padding(8)
+                .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                createVersionButton
+                switchVersionButton
+            }
+        }
+        .errorAlert(presentingError: $error)
+    }
+    
+    /// The button for editing the damage type of the selected feature.
+    private var editDamageTypeButton: some View {
+        Button("Edit", systemImage: "pencil") {
+            isSettingDamageType = true
+        }
+        .imageScale(.large)
+        .labelStyle(.iconOnly)
+        .disabled(model.onDefaultVersion)
+        .confirmationDialog("Damage Type", isPresented: $isSettingDamageType, titleVisibility: .visible) {
+            ForEach(DamageType.allCases, id: \.self) { damageType in
+                Button(damageType.rawValue) {
```

Comment:
Similar to `VersionAccess`, it would be better not to use the case name (non-localized) as a user-facing (localized) string as it isn't best practice for a production app.